src/views/virtualmachinesinstances/list/VirtualMachinesInstancesRow.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import { VMStatusConditionLabelList } from 'src/views/virtualmachines/list/components/VMStatusConditionLabel';
+
+import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { ResourceLink, RowProps, TableData } from '@openshift-console/dynamic-plugin-sdk';
+
+import VirtualMachinesInstancesStatus from './VirtualMachinesInstancesStatus';
+
+type VirtualMAchinesInstancesRowProps = RowProps<V1VirtualMachineInstance, { kind: string }>;
+
+const VirtualMAchinesInstancesRow: React.FC<VirtualMAchinesInstancesRowProps> = ({
```

Comment:
`VirtualMachinesInstancesRow`, `VirtualMachinesInstancesRowProps`

Reply:
u want to be my code speller? u better in this than this extension :) changed, thanks!